azdev/params.py

```diff
@@ -34,16 +34,20 @@ def load_arguments(self, _):
         c.argument('git_repo', options_list='--repo', arg_group='Git', help='Path to the Git repo to check.')
 
     with ArgumentsContext(self, 'setup') as c:
-        c.argument('cli_path', options_list=['--cli', '-c'], nargs='?', const=Flag, help="Path to an existing Azure CLI repo. Omit value to search for the repo or use special value 'EDGE' to install the latest developer edge build.")
-        c.argument('ext_repo_path', options_list=['--repo', '-r'], nargs='+', help='Space-separated list of paths to existing Azure CLI extensions repos.')
```

Comment:
azdev is supposed to support multiple extension repos, they could be seperated folder. 
Is it necessary to modify this?

Reply:
First, what is the use case for multiple extensions repos? It is a git repository, why can't a user switch between branches, forks, and tags like one would normally do? 
Moreover, the CLI can only "look" for one extensions repo via the azure config file, same with the codgen. Also, multiple repos risk breaking the virtual venv isolation, because again the azure CLI can only look for one extensions repo. If you allow multiple to be changed with in one azure dev environment, on of those repos could be reference by another CLI in its azure config and cause unexpected behavior.